bootstrap/venv.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh -e
+#
+# Installs and updates letencrypt virtualenv
+#
+# USAGE: source ./dev/venv.sh
+
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-~/.local/share}
+VENV_NAME="letsencrypt"
+VENV_PATH=${VENV_PATH:-"$XDG_DATA_HOME/$VENV_NAME"}
+
+# virtualenv call is not idempotent: it overwrites pip upgraded in
+# later steps, causing "ImportError: cannot import name unpack_url"
+if [ ! -d $VENV_PATH ]
+then
+  virtualenv --no-site-packages --python python2 $VENV_PATH
+fi
+
+. $VENV_PATH/bin/activate
+pip install -U setuptools
+pip install -U pip
+
+# TODO: install apache and nginx plugins by default?
+# --pre is not necessary for dev releases in more recent pip versions
+pip install -U letsencrypt letsencrypt-apache letsencrypt-nginx
```

Comment:
we decided not to install `letsencrypt-nginx`

Are you going to `source` this script from the `letsencrypt-auto` wrapper?